test/parallel/test-stream-reduce.js

```diff
@@ -121,6 +121,8 @@ function sum(p, c) {
   // Error cases
   assert.rejects(() => Readable.from([]).reduce(1), /TypeError/);
   assert.rejects(() => Readable.from([]).reduce('5'), /TypeError/);
+  assert.rejects(() => Readable.from([]).reduce((x, y) => x + y, 0, 1), /ERR_INVALID_ARG_TYPE/);
+  assert.rejects(() => Readable.from([]).reduce((x, y) => x + y, 0, { signal: true }), /ERR_INVALID_ARG_TYPE/);
```

Comment:
these two should probably have `.then(common.mustCall())` (e.g. like you did in test-stream-some-every)

Reply:
You can also add them to the two lines above them (that you didn't add).